Airports/EGFE/Runway.txt

```diff
@@ -0,0 +1,2 @@
+03 21  029 209 N051.49.44.810 W004.57.52.950 N051.50.14.100 W004.57.25.990
```

Comment:
Noted: RWY Headings rounded down rather than up, but makes no difference to anything as far as I know.

Reply:
@hazzas-99 Oh! I was virtually certain it was the other way round. Although I can't find any documentation about it either way now I come to look. And I thought someone told me these numbers weren't used for anything by euroscope, do you know what they're used for out of interest?